staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -151,11 +212,112 @@ func WithTypeAndObjectMeta(s *schema.Structural) *schema.Structural {
 	props["metadata"] = schema.Structural{
 		Generic: schema.Generic{Type: "object"},
 		Properties: map[string]schema.Structural{
-			"name": stringType,
+			"name":         stringType,
 			"generateName": stringType,
 		},
 	}
 	result.Properties = props
 
 	return result
-}
\ No newline at end of file
+}
+
+// estimateMinSizeJSON estimates the minimum size in bytes of the given schema when serialized in JSON.
+// minLength/minProperties/minItems are not currently taken into account, so if these limits are set the
+// minimum size might be higher than what estimateMinSizeJSON returns.
+func estimateMinSizeJSON(s *schema.Structural) int64 {
```

Comment:
might consider a safe-guard like this in case a nil schema finds its way here

```go
if s == nil {
  return 1
}
```

Reply:
Gotcha, makes sense - I'll mark this as resolved once I add a nil check.

Reply:
did this get added? I'm not seeing it

Reply:
Whoops! The commit should be pushed now.